Xamarin.Essentials/Sms/Sms.android.cs

```diff
@@ -24,16 +25,15 @@ static Task PlatformComposeAsync(SmsMessage message)
         }
 
         static Intent CreateIntent(SmsMessage message)
-            => CreateIntent(message?.Recipient, message?.Body);
+            => CreateIntent(message?.Recipients, message?.Body);
 
-        static Intent CreateIntent(string recipient, string body = null)
+        static Intent CreateIntent(string[] recipients, string body = null)
         {
             Intent intent = null;
 
             body = body ?? string.Empty;
-            recipient = recipient ?? string.Empty;
 
-            if (string.IsNullOrWhiteSpace(recipient) && Platform.HasApiLevel(BuildVersionCodes.Kitkat))
+            if (recipients.All(x => string.IsNullOrWhiteSpace(x)) && Platform.HasApiLevel(BuildVersionCodes.Kitkat))
```

Comment:
let's swap these calls and check kitkat first. Small optimization

Reply:
Done